### ISSUE TITLE
Column edit

### ISSUE THREAD

**Issue by ags1234** [notepad-plus-plus/notepad-plus-plus#13799]
I have a text file, with lines of variable length. I want to insert characters on the end of some lines.
In CudaText, if the lines are one after the other, I press alt and drag with left-click over the lines. If the lines are not consecutive, I press ctrl and left-click over those specific lines. Pretty intuitive.
How can I do that in Notepad++?

**Comment by alankilborn** [comment]
>I wonder why is this code not included in Notepad++ by default.

Scintilla is Notepad++'s editing component (and a separate project).
Scintilla _may_ support these things, they may just have to be "enabled" by Notepad++.
If you're inclined, you can research that and post info here about how to enable, and the Notepad++ developers may listen and integrate such enablers.

**Comment by ags1234** [comment]
@alankilborn 

Hey, thanks! Following your advice, I tried to read some of the Scintilla documentation. What I found was too technical for me, details about functions written in a programming language.
Too bad, because Notepad++ seems like the perfect text editor.
Where it lacks from my point of view is multi-caret editing (the plugin covers that) and custom syntax highlighting using regulated expressions.

Another editor I tried, CudaText, has both of those without the need of plugins. It is also very well maintained by it's author. But it doesn't know how to print files 😕

**Comment by alankilborn** [comment]
With Notepad++, functionality that is currently available via a plugin rarely goes into the native/standard product.  The philosophy is: Use the plugin.

>Where it lacks from my point of view is multi-caret editing (the plugin covers that)

Not sure what everyone finds so fascinating about multi-caret...
There's a related discussion going on [HERE](https://community.notepad-plus-plus.org/topic/24549/automatic-selection-of-all-instances-of-a-searched-string-in-one-go).

>and custom syntax highlighting using regulated expressions.

Notepad++ has UDL and there's a plugin called _EnhanceAnyLexer_...

**Comment by rdipardo** [comment]
> [...] there's a plugin called *EnhanceAnyLexer* . . .

You may think the name *EnhanceAnyLexer* is self-explanatory because you already know about its regex capabilities.
But put yourself in the shoes of a casual end user, and at least share a link: https://github.com/Ekopalypse/EnhanceAnyLexer#readme

**Comment by alankilborn** [comment]
>But put yourself in the shoes of a casual end user, and at least share a link

Nah. It's the first search engine hit.  If someone is even remotely intrigued, they can "google" it, rather than me going to the effort of link-izing it.
